build.assets/genproto.sh

```diff
@@ -14,7 +14,8 @@ main() {
   rm -fr api/gen/proto gen/proto  # cleanup gen/proto folders
 
   # Generate Gogo protos (default).
-  buf generate --template=buf-gogo.gen.yaml
+  buf generate --template=buf-gogo.gen.yaml api/proto
+  buf generate --template=buf-gogo.gen.yaml proto
```

Comment:
```suggestion
  buf generate --template=buf-gogo.gen.yaml \
    --path=api/proto \
    --path=proto
```

Reply:
This won't work unfortunately, as I mentioned in my other comment.

Reply:
Thanks for trying.